server/prisma/schema.prisma

```diff
@@ -10,26 +10,95 @@ datasource db {
 
 // This comes with the boilerplate.
 model User {
-  id                  String          @id @default(cuid())
-  createdAt                        DateTime        @default(now())
-  updatedAt                        DateTime        @updatedAt
-  email           String  @unique
-  password      String
-  role          String
-  lastLogin     DateTime?  
-  // LL: https://www.smashingmagazine.com/2017/11/safe-password-resets-with-json-web-tokens/
-  resetPasswordToken String
+  id                       String             @id @default(cuid())
+  createdAt                DateTime           @default(now())
+  updatedAt                DateTime           @updatedAt
+  email                    String             @unique
+  password                 String
+  role                     String
+  lastLogin                DateTime?
+  resetPasswordToken       String
   dateResetPasswordRequest DateTime?
-  validateEmailToken String
-  isEmailValidated Boolean
-  name  String
+  validateEmailToken       String
+  isEmailValidated         Boolean
+  name                     String
+  feeds                    Feed[] // one to many
+  articles                 ArticlesOnUsers[]
+  settings                 Settings?
+  subscriptionPlans        SubscriptionPlan[]
 }
 
 model Feed {
+  id        String               @id @default(cuid())
+  title     String
+  createdAt DateTime             @default(now())
+  updatedAt DateTime             @updatedAt
+  deletedAt DateTime? //admin could delete
```

Comment:
change comment to specify that this is a soft delete mechanism. otherwise it's confusing, admin could delete but you're not actually deleting the row in the database